docroot/themes/humsci/humsci_basic/src/scss/partials/admin/_user-login-page.scss

```diff
@@ -0,0 +1,179 @@
+  background-repeat: no-repeat;
+
+  .block--page-title {
+    display: none;
+  }
+}
+
+.user-login-form {
+  background-color: var(--palette--white);
+  padding: hb-calculate-rems(24px);
+  max-width: hb-calculate-rems(900px);
+  width: auto;
+  margin: hb-calculate-rems(24px);
+  margin-bottom: hb-calculate-rems(24px) !important;
+
+  @include grid-media-min('md') {
+    width: fit-content;
+    margin: 0 auto;
+    margin-bottom: 0 !important;
+    padding: hb-calculate-rems(24px) hb-calculate-rems(32px);
+  }
+
+  @include grid-media-min('lg') {
+    padding: hb-calculate-rems(24px) hb-calculate-rems(48px);
+  }
+
+  div[data-drupal-selector="edit-intro-text"] h1 {
+    display: block !important;
+    font-weight: hb-theme-font-weight(semibold);
+    font-size: hb-calculate-rems(28px);
+    line-height: hb-calculate-rems(38px);
+    margin: 0;
+
+    @include grid-media-min('md') {
+      font-size: hb-calculate-rems(30px);
+    }
+  }
+
+  h4,
+  details[open].form-wrapper summary {
+    margin-top: hb-calculate-rems(24px);
+    margin-bottom: hb-calculate-rems(22px);
+    font-weight: hb-theme-font-weight(semibold);
+    font-size: hb-calculate-rems(21px);
+    line-height: 127%;
+
+    @include grid-media-min('md') {
+      font-size: hb-calculate-rems(22px);
+    }
+  }
+
+  a.samlauth-login {
+    @include hb-big-button;
+
+    max-width: hb-calculate-rems(197px);
+    margin-top: hb-calculate-rems(26px);
+    background-color: var(--palette--cardinal);
+    border-color: var(--palette--cardinal);
+
+    &::after {
+      content: none;
+    }
+  }
+
+  &:has(details.form-wrapper) {
+    @include grid-media-min('md') {
+      width: auto;
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      align-items: start;
+      margin: hb-calculate-rems(24px);
+      margin-bottom: hb-calculate-rems(24px) !important;
+
+      div[data-drupal-selector="edit-intro-text"],
+      a.samlauth-login {
+        grid-column: 1;
+      }
+
+      > details.form-wrapper {
+        grid-column: 2;
+        grid-row: 6 / -1;
+      }
+
+      div[data-drupal-selector="edit-intro-text"] {
+        grid-row: 2;
+      }
+
+      a.samlauth-login {
+        grid-row: 3;
+      }
+    }
+
+    @include grid-media-min('lg') {
+      margin: 0 auto;
+      margin-bottom: 0 !important;
+    }
+  }
+
+  details[open].form-wrapper {
+    margin: hb-calculate-rems(32px) hb-calculate-rems(-24px) hb-calculate-rems(-24px);
+    padding: hb-calculate-rems(24px);
+    background-color: var(--palette--secondary-highlight);
+    
+    @include grid-media-min('md') {
+      margin: hb-calculate-rems(-24px) hb-calculate-rems(-32px) hb-calculate-rems(-24px) 0;
+      padding-right: hb-calculate-rems(32px);
+    }
+
+    @include grid-media-min('lg') {
+      margin-right: hb-calculate-rems(-48px);
+      padding-right: hb-calculate-rems(48px);
+      padding-left: hb-calculate-rems(45px);
+    }
+
+    summary {
+      background-color: transparent;
```

Comment:
Please add `pointer-events: none;` here to ensure that the accordion cannot be closed.